bon/tests/integration/builder_on_fn.rs

```diff
@@ -57,17 +58,18 @@ fn default_attr() {
     fn sut(
         #[builder(default)] arg1: u32,
         #[builder(default = 42)] arg2: u32,
-        #[builder(default = "default")] arg3: String,
-        #[builder(default = vec![42])] arg4: Vec<u32>,
-    ) -> (u32, u32, String, Vec<u32>) {
+        #[builder(default = "default")] arg3: &str,
```

Comment:
This should also stay under `alloc`. It validates that `default` close does `into` conversion on its expression. Alterantively, these last two arguments could be separated into a different test under `alloc`, to keep the test for the first two arguments no_std-compatible

Reply:
Done